src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/ContentDbContextModelSnapshot.cs

```diff
@@ -1503,9 +1503,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                             ContactId = new Guid("74f5aade-6d24-4a0b-be23-2ab4b4b2d191"),
                             LegacyPublicationUrl = "https://www.gov.uk/government/collections/statistics-school-applications",
                             MethodologyId = new Guid("8ab41234-cc9d-4b3d-a42c-c9fce7762719"),
-                            Slug = "secondary-and-primary-schools-applications-and-offers",
+                            Slug = "secondary-and-primary-school-applications-and-offers",
```

Comment:
This might be problematic as we dont have a redirect system currently

Reply:
There hasn't been any Releases of this Publication yet.